tokens/src/tokens/alias/elevation-border.json

```diff
@@ -0,0 +1,11 @@
+{
+  "alias": {
+    "elevation": {
```

Comment:
Why have we created this file? the color border doesn't belong to category 'elevation'. It belongs to 'color' category

Reply:
Changed as we talked by Slack